### ISSUE TITLE
Test are not passing

### ISSUE THREAD

**Issue by Andreyco** [poppinss/indicative#143]
```
1. should be able to use rule values as argument
  expected 'age should be over {{argument.0}} and under {{argument.1}}' to equal 'age should be over 18 and under 40'
  "age should be over {{argument.0}} and under {{argument.1}}" => "age should be over 18 and under 40"
```

**Comment by Andreyco** [comment]
External `pope@1.0.3` breaks tests.
#144 fixes this issue by locking to specific version.

**Comment by Andreyco** [comment]
External `pope@1.0.3` breaks tests.
#144 fixes this issue by locking to specific version.

cc @thetutlage

**Comment by thetutlage** [comment]
Fixed with release of `1.0.4` in pope
